lib/src/kbasesearchengine/tools/SearchTools.java

```diff
@@ -664,6 +679,13 @@ private void printError(final String msg) {
                 "Start an indexer worker with the provided id. At any given time, all workers " +
                 "MUST have unique ids. Set the id to '-' to generate a random id.")
         private String startWorker;
+
+        // TODO SCT-398 It's pretty clear that many of the errors we're seeing (too many features/contigs, location missing) are pointless to re-run, so we're going to want to be able to select failed events with more precision.
```

Comment:
A simple future improvement could be specifying the type of error(s) to be rerun. That'd probably cover 90% of the necessary granularity